### PR TITLE
fix(tests): Fix the failing functional test.

### LINKED ISSUE #3704: Failing functional test: "signup - signup with existing account, coppa is valid, credentials are wrong"
> ✓ firefox on any platform - signup - signup with existing account, coppa is valid, credentials are correct (0.969s)
× firefox on any platform - signup - signup with existing account, coppa is valid, credentials are wrong (29.84s)
NoSuchElement: [POST http://localhost:4444/wd/hub/session/46328520-05d5-447f-962c-b3fbc66fdbe8/element / {"using":"css selector","value":"#fxa-signin-header"}] Unable to locate element: {"method":"css selector","selector":"#fxa-signin-header"}
For documentation on this error, please visit: http://seleniumhq.org/exceptions/no_such_element.html
Build info: version: '2.53.0', revision: '35ae25b', time: '2016-03-15 17:00:58'
System info: host: 'Shanes-Retina-MBP.local', ip: '10.246.29.79', os.name: 'Mac OS X', os.arch: 'x86_64', os.version: '10.11.4', java.version: '1.7.0_40'
Driver info: driver.version: unknown
  at runRequest  <node_modules/intern/node_modules/leadfoot/Session.js:88:40>
  at <node_modules/intern/node_modules/leadfoot/Session.js:109:39>
  at new Promise  <node_modules/intern/node_modules/dojo/Promise.ts:411:3>
  at ProxiedSession._post  <node_modules/intern/node_modules/leadfoot/Session.js:63:10>
  at ProxiedSession.Session.find  <node_modules/intern/node_modules/leadfoot/Session.js:1125:15>
  at Command.<anonymous>  <node_modules/intern/node_modules/leadfoot/Command.js:42:36>
  at <node_modules/intern/node_modules/dojo/Promise.ts:393:15>
  at run  <node_modules/intern/node_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/node_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:419:9>
  at Command.find  <node_modules/intern/node_modules/leadfoot/Command.js:23:10>
  at Command.prototype.(anonymous function) [as findByCssSelector]  <node_modules/intern/node_modules/leadfoot/lib/strategies.js:24:16>
  at Test.registerSuite.signup with existing account, coppa is valid, credentials are wrong [as test]  <tests/functional/sign_up.js:303:10>
  at <node_modules/intern/lib/Test.js:181:24>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:393:15>
  at runCallbacks  <node_modules/intern/browser_modules/dojo/Promise.ts:11:11>
  at <node_modules/intern/browser_modules/dojo/Promise.ts:317:4>
  at run  <node_modules/intern/browser_modules/dojo/Promise.ts:237:7>
  at <node_modules/intern/browser_modules/dojo/nextTick.ts:44:3>
  at nextTickCallbackWith0Args  <node.js:419:9>

STR: 

1. Run `npm run test-functional grep="signup with existing account, coppa is valid"`

### PR DESCRIPTION
**Pull request by shane-tomlinson**
Fix failing functional tests when run w/ Selenium 2.53.0:

1. "signup with existing account, coppa is valid, credentials are wrong"
1. "delete_account - locked account, verify different browser - from original
   tab's P.O.V."
1. "settings->change password with verified email - locked account, verify
   different browser - from new browser's P.O.V."

In `visibleByQSA` ensure the element is both visible and not animating. Remove
`visibleByQSAErrorHeight`.

fixes #3704 

@vbudhram or @vladikoff - mind an r?

### CODE REVIEW

File: tests/functional/lib/helpers.js [hunk 1]
```diff
@@ -173,32 +173,42 @@ define([
...
+        throw new Error('Multiple elements matched. Make a more precise selector - ' + selector);
+      }
+
+      if (! $matchingEl.is(':visible')) {
+        return null;
+      }
 
-  function visibleByQSAErrorHeight(selector) {
-    return visibleByQSA(selector, {offsetHeight: 10});
+      if ($matchingEl.is(':animated')) {
```
Comment (vladikoff) [inline tests/functional/lib/helpers.js]: as long as we use jQuery for animations we should be good here

**Comment by shane-tomlinson** [comment]
`oauth signup - signup with existing account, coppa is valid, credentials are wrong` still fails. Updating

**Comment by vbudhram** [comment]
Tests are still failing

```bash
Created session firefox on any platform (54f0e929-6a30-411e-bb21-81f4fbeb6f40)

fxa-content-server.route.500.ERROR: TypeError: Object.keys called on non-object

    at Function.keys (native)

    at reportError (/home/travis/build/mozilla/fxa-content-server/server/lib/routes/post-metrics-errors.js:58:37)

    at process (/home/travis/build/mozilla/fxa-content-server/server/lib/routes/post-metrics-errors.js:98:7)

    at Layer.handle [as handle_request] (/home/travis/build/mozilla/fxa-content-server/node_modules/express/lib/router/layer.js:95:5)

    at next (/home/travis/build/mozilla/fxa-content-server/node_modules/express/lib/router/route.js:131:13)

    at Route.dispatch (/home/travis/build/mozilla/fxa-content-server/node_modules/express/lib/router/route.js:112:3)

    at Layer.handle [as handle_request] (/home/travis/build/mozilla/fxa-content-server/node_modules/express/lib/router/layer.js:95:5)

    at /home/travis/build/mozilla/fxa-content-server/node_modules/express/lib/router/index.js:277:22

    at Function.process_params (/home/travis/build/mozilla/fxa-content-server/node_modules/express/lib/router/index.js:330:12)
```

**Comment by vladikoff** [comment]
> Tests are still failing

This is unrelated, you might need to run `bower install`

**Comment by vbudhram** [comment]
Oh this was in Travis, it is strange that it still passed the build?

https://travis-ci.org/mozilla/fxa-content-server/jobs/127799377

**Comment by vladikoff** [comment]
@vbudhram oh didn't notice that one. seems like some bad input. It's not strange that it passed because in some cases we expect the error responses from the server. I'll open an issue to fix that

**Pull request merged**